packages/runtime/plugin-runtime/package.json

```diff
@@ -93,6 +93,11 @@
       "jsnext:source": "./src/router/runtime/server.ts",
       "default": "./dist/esm/router/runtime/server.js"
     },
+    "./router/cli": {
```

Comment:
Where did we use `@modern-js/runtime/router/cli`?

Reply:
in inside, router plugin need use router function. The export here can be removed after the plugin modifications are completed.

Reply:
Ok, i see it~